test/active_record_doctor/detectors/incorrect_length_validation_test.rb

```diff
@@ -61,6 +61,40 @@ def test_no_validation_and_no_limit_is_ok
     refute_problems
   end
 
+  def test_array_inclusion_validation_with_shorter_values_and_limit_is_ok
+    Context.create_table(:users) do |t|
+      t.string :status, limit: 64
+    end.define_model do
+      validates :status, inclusion: { in: ["new", "vip"] }
```

Comment:
I suggest including the longest lenght that will work - 64 in this case - to test the boundary.